### DISCUSSION TITLE
Couldn't find a file error on files with # in path

### DISCUSSION THREAD

**Discussion by sfairat15** [community/community#9128] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello!

**Context**
I have several files at the repo at directory with # in its name, for example `dev-cc#RU`.
Code Search shows files at this folder at search results.

**Problem**
When I try to open file from search results, clicking on the link `Some/Path/dev-cc#RU/connectionStrings.transform.config`, I get an error `Couldn't find a file called Some/Path/dev-cc`

**Expected**
Code Search opens file content

P.S. It become more confusing, if there are only one file at search result, because it redirects to the file instantly, see #8622

**Comment by look** [comment, marked as answer]
Thanks for the report @sfairat15! I've just deployed a fix for this.
